docs/src/snap/howto/install/install-offline.md

```diff
@@ -0,0 +1,337 @@
+   push to your registry mirror.
+3. Configure the Canonical Kubernetes container runtime (`containerd`) to load
+   images from
+   the private registry mirror instead of the upstream source. This will be
+   described in the [Configure registry mirrors](
+      #Container-Runtime-Option-B:-Configure-registry-mirrors) section.
+
+In order to load images into the private registry, you need a machine with
+access to both any upstream registries (e.g. `docker.io`)
+and the private mirror.
+
+##### Load images with regsync
+
+We recommend using [regsync][regsync] to copy images
+from the upstream registry to your private registry.
+For the images used in the k8s-snap we currently sync upstream images
+to the `ghcr.io` repo.
+For a similar task, it may be helpful to refer to the
+[upstream-images.yaml][upstream-imgs] file as well as the
+[sync-images][sync-images] script.
+
+Using the output from `sudo k8s list-images`,
+create a yaml file with the images to sync to the private registry.
+
+<!-- markdownlint-disable MD013 -->
+```yaml
+# my-images.yaml
+sync:
+  - source: ghcr.io/canonical/k8s-snap/pause:3.10
+    target: '{{ env "MIRROR" }}/canonical/k8s-snap/pause:3.10'
+    type: image
+  - source: ghcr.io/canonical/cilium-operator-generic:1.15.2-ck2
+    target: '{{ env "MIRROR" }}/canonical/cilium-operator-generic:1.15.2-ck2'
+    type: image
+  - source: ghcr.io/canonical/cilium:1.15.2-ck2
+    target: '{{ env "MIRROR" }}/canonical/cilium:1.15.2-ck2'
+    type: image
+  - source: ghcr.io/canonical/coredns:1.11.1-ck4
+    target: '{{ env "MIRROR" }}/canonical/coredns:1.11.1-ck4'
+    type: image
+  - source: ghcr.io/canonical/k8s-snap/sig-storage/csi-node-driver-registrar:v2.10.1
+    target: '{{ env "MIRROR" }}/canonical/k8s-snap/sig-storage/csi-node-driver-registrar:v2.10.1'
+    type: image
+  - source: ghcr.io/canonical/k8s-snap/sig-storage/csi-provisioner:v5.0.1
+    target: '{{ env "MIRROR" }}/canonical/k8s-snap/sig-storage/csi-provisioner:v5.0.1'
+    type: image
+  - source: ghcr.io/canonical/k8s-snap/sig-storage/csi-resizer:v1.11.1
+    target: '{{ env "MIRROR" }}/canonical/k8s-snap/sig-storage/csi-resizer:v1.11.1'
+    type: image
+  - source: ghcr.io/canonical/k8s-snap/sig-storage/csi-snapshotter:v8.0.1
+    target: '{{ env "MIRROR" }}/canonical/k8s-snap/sig-storage/csi-snapshotter:v8.0.1'
+    type: image
+  - source: ghcr.io/canonical/metrics-server:0.7.0-ck0
+    target: '{{ env "MIRROR" }}/canonical/metrics-server:0.7.0-ck0'
+    type: image
+  - source: ghcr.io/canonical/rawfile-localpv:0.8.0-ck5
+    target: '{{ env "MIRROR" }}/canonical/rawfile-localpv:0.8.0-ck5'
+    type: image
+```
+<!-- markdownlint-enable MD013 -->
+
+After you have updated the yaml file, use [regctl][regctl] to sync the images.
+Run the [sync-images][sync-images] script in `./build-scripts/hack`:
+
+```bash
+USERNAME="$username" PASSWORD="$password" MIRROR="$mirror" ./sync-images.sh \
+once -c ./my-images.yaml
+```
```

Comment:
this is not correct. first, the correct tool is `regsync` not `regctl`

Secondly, the `sync-images.sh` script syncs from `upstream-images.yaml` in the repo. Instead, show how people can use regsync with their custom yaml:

Assuming your registry mirror is at http://10.10.10.10:5050

```
USERNAME="$username" PASSWORD="$password" MIRROR="10.10.10.10:5050" ./src/k8s/tools/regsync.sh once -c ./my-images.yaml
```